### PR TITLE
Jtm subsidy rounding bug

### PR DESCRIPTION
**Pull request by BigChiefSmidgeums**

### CODE REVIEW

File: app/controllers/subsidies_controller.rb [hunk 1]
```diff
@@ -23,7 +23,9 @@ class SubsidiesController < ApplicationController
 
   def create
     @sub_service_request = SubServiceRequest.find params[:subsidy][:sub_service_request_id]
-    @subsidy = PendingSubsidy.create(sub_service_request_id: @sub_service_request.id, percent_subsidy: 0)
+    @subsidy = PendingSubsidy.new(sub_service_request_id: @sub_service_request.id)
+    @subsidy.percent_subsidy = @subsidy.default_percentage
+    @subsidy.save
```
Comment (wtholt) [inline app/controllers/subsidies_controller.rb]: This is going to clash with #614

**Comment by amcates** [comment]
This is breaking one spec, rspec ./spec/features/service_subsidy_spec.rb:51 # subsidy page Adding a subsidy should adjust the pi contribution if a subsidy percentage is added

File: spec/features/service_subsidy_spec.rb [hunk 2]
```diff
@@ -56,7 +56,7 @@
       wait_for_javascript_to_finish
       adjusted_pi_contribution = @direct_cost - (@direct_cost * percent_subsidy)
       pi_field_value = find('#pi_contribution').value.gsub(/,/, '')
-      expect(pi_field_value).to eq('$' + adjusted_pi_contribution.to_s)
+      expect(pi_field_value).to eq('$' + adjusted_pi_contribution.to_s + '0')
```
Comment (amcates) [inline spec/features/service_subsidy_spec.rb]: And why are you adding  '0' string to this?  I would think the adjusted_pi_contribution calculation would be correct.  This seems hacky to me.

**Comment by jleonardw9** [comment]
Looks good.

**Comment by jayhardee9** [comment]
Looks good

**Pull request merged**